docs/general/events.md

```diff
@@ -32,17 +32,73 @@ field semantics, and stability and requirement levels. Other events may be user-
+* It MAY contain a _payload_ (data) that describes the specific details of the
+  named event.
+* The structure of the named event (`event.name`) is unique to the event, this
+  identifies
+  * Whether the event contains a _payload_ (data), and the structure / type of
+    the _payload_ (data).
+  * Whether there are any expected [standard](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/common#attribute)
+    attributes that are used to describe the event.
+* All OpenTelemetry defined events SHOULD be defined in the semantic conventions
+  to highlight the expected structure of the event.
+* The _payload_ (data) MAY contain any type supported by the OpenTelemetry data
+  model for the log [body](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#field-body)
+  and the semantic conventions will define the expected structure of the _payload_
+  (data) for the event.
+* The _payload_ (data) SHOULD be used to represent the details.
+* Events with the same `event.name` are structurally similar to one another and
+  MAY (depending on the event) be comparable.
+
+Recommendations for defining events:
+
+* Use the _payload_ (data) to represent the details of the event instead of a
+  collection of  over [standard](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/common#attribute)
+  attributes.
+* Generators of events SHOULD use the [Event API](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/event-api.md)
+  to create and emit events.
+  * While the data model is based on `LogRecord`, it should NOT be assumed that
+    ALL consumers will process events as `LogRecords`, some may choose to provide
+    their own Event API implementation.
+* The _payload_ (data) _fields_ SHOULD not be prefixed with the `event.name` to
+  avoid redundancy and to keep the event definition clean.
 
-## General event attributes
+### Event payload (data)
 
-Events are recorded as LogRecords that are shaped in a special way: Event
-LogRecords MUST have the attribute `event.name` that uniquely identifies the event.
-Events with the same `event.name` are structurally similar to one another. Events
-may also have other LogRecord attributes.
+* The yaml definition for OpenTelemetry defined events will support describing
+  individual _fields_
+  * Any _fields_ are NOT added to the Global attribute registry for use by other
+    events and are NOT added to the Global attribute registry.
+  * The _fields_ are unique to the named event (`event.name`) and different events
+    may use the same _field_ name to represent different data, due to the unique
+    nature of the event.
+* The _fields_ MAY reference / inherit details from Global attribute registry
+  attributes and provide additional details specific to the event, including
+  providing an _alias_ (shorter) name for the attribute.
```

Comment:
not blocking: it's not implemented today and I find it confusing having spec in a future tense. Given we already documented that event payload fields are NOT OTel attributes, I'd refer to remove this paragraph or replace it with something like "Common attribute naming conventions and [registry](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/README.md) requirements don't apply to event payload fields."